packages/core/config/webpack.common.js

```diff
@@ -314,7 +315,7 @@ module.exports = (webpackEnv) => {
       runtimeChunk: {
         name: (entrypoint) => `runtime-${entrypoint.name}`,
       },
-      minimizer: [isEnvProduction && new CssMinimizerPlugin()].filter(Boolean),
+      minimizer: [new CssMinimizerPlugin(), new TerserPlugin()],
```

Comment:
Could this lead to a performance impact during development? 🤷‍♂️

Reply:
This is what I thought originally as well (and had the env check)

But looking further into the docs there was this one line in the TerserPlugin docs: 
> Since version 4 webpack runs optimizations for you depending on the chosen mode, still all optimizations are available for manual configuration and overrides.

And the docs here: https://webpack.js.org/configuration/mode/

But webpack is confusing and maybe I'm wrong 😑. 

So what I think happens is in mode dev, minimize gets set to false so these plugins never get run. 

Is there a good way to verify this?

Reply:
Hmpf, that is super confusing... they're run based on the mode, but they're also configurable manually? I wonder what they mean by that... I think _we are_ configuring them manually here.

I guess you could see if the output bundles are minified in development mode, which would indicate that it's running when it shouldn't be.
![image](https://user-images.githubusercontent.com/44849/114396061-66b15300-9b9d-11eb-95d3-e8ec84687b4f.png)

Reply:
I don't know if source maps are applied to the response 🤷

Reply:
Hey @peterp can confirm no minimising on the dev server:
![image](https://user-images.githubusercontent.com/1521877/114427808-5ad08b80-9bb3-11eb-87a2-173488472154.png)

To confirm I've linked the project correctly:
```terminal
❯ cat node_modules/@redwoodjs/core/config/webpack.common.js | grep minimizer
const CssMinimizerPlugin = require('css-minimizer-webpack-plugin')
      minimizer: [new CssMinimizerPlugin(), new TerserPlugin()],
```

Reply:
My understanding of why this works btw:
1. When `mode` isn't production, webpack automatically sets the flag `minimize` (not _not_ minimize**r**) to false
2. When `optimizations.minimize` is false, `optimizations.minizer` is ignored

Reply:
Nice, could you leave a comment nearby `minimizer ` so that we don't assume that it's running in dev mode in the future?

Reply:
You bet ☑️